traction_hw:
  verbose: False
  sampling_freq: 100  # Hz
  joints:
    right_wheel:
      spi_dev: 0
    left_wheel:
      spi_dev: 1
